.github/workflows/add-new-issue-to-project.yml

```
@@ -0,0 +1,18 @@
+name: 🤖Add New Issue To Project
+
+
+on:
+  issues:
+    types: opened
+
+
+jobs:
+  add_new_issue_to_project:
+    name: Add New Issue
+    uses: KinsonDigital/Infrastructure/.github/workflows/add-issue-to-project.yml@v8.2.0
```

Update to latest version:

```suggestion
    uses: KinsonDigital/Infrastructure/.github/workflows/add-issue-to-project.yml@v9.0.0
```